ipyaladin/aladin_widget.py

```diff
@@ -208,7 +233,13 @@ def add_table(self, table):
                 if isinstance(item, bytes):
                     row_data.append(item.decode('utf-8'))
                 else:
+                    if not isinstance(item, str):
+                        # replace NaN by " ", this is a quick fix 
+                        # and should be questioned when doing a rework
+                        # of this function
+                        item = " " if math.isnan(item) else item 
                     row_data.append(item)
+                    
```

Comment:
I do not like it either but if it solves issues without so much performance cost, I am for it.

I just ask myself if it would not be possible to do that from the numpy array table_array directly instead of looping in python over all the columns and then all the rows.